src/org/zaproxy/zap/extension/jython/ExtensionJython.java

```diff
@@ -50,11 +53,24 @@ public ExtensionJython() {
 	public void hook(ExtensionHook extensionHook) {
 		super.hook(extensionHook);
 
+		this.jythonOptionParam = new JythonOptionParam();
+		extensionHook.addOptionsParamSet(this.jythonOptionParam);
+		if(View.isInitialised()){
+			extensionHook.getHookView().addOptionPanel(new JythonOptionPanel());
+		}
+	}
+	
+	@Override
+	public void start() {

```

Comment:
Why `start`? If it's because of the options it could be done in `optionsLoaded` instead (which is done sooner and immediately after the call to `hook` method).